great_expectations/rule_based_profiler/parameter_builder/parameter_container.py

```diff
@@ -140,21 +160,18 @@ def build_parameter_container(
     :param parameter_values
     Example of required structure for "parameter_values" (matching the type hint in the method signature):
```

Comment:
The word "required" is throwing me here. Is this structure actually required?

Reply:
@talagluck Changed to emphasize that this is just name-value pairs.